scripts/script_backup_db_data.sh

```diff
@@ -0,0 +1,21 @@
+if [ -n "$QTREES_VERSION" ]; then
+  now=$(date +"%m_%d_%Y")
+  if [ $# -eq 0 ]; then
+    # default dir
+    data_dir="data/db/"$QTREES_VERSION
+  else
+    # given dir as first parameter
+    data_dir=$1"/"$QTREES_VERSION
+  fi
+  # create dir in case of need
+  mkdir -p $data_dir
+  filename=$data_dir"/dump_$now.sql"
```

Comment:
Das ist ja irgend nen binary format. Daher ggf. einfach ne andere Änderung? 
Die [Dokumentation](https://www.postgresql.org/docs/current/app-pgdump.html) schlägt ".dump" for.

Reply:
done